File: app/src/main/java/se/gustavkarlsson/skylight/android/background/providers/DarknessProvider.java
package se.gustavkarlsson.skylight.android.background.providers;

import org.threeten.bp.Instant;

import se.gustavkarlsson.skylight.android.models.factors.Darkness;

public interface DarknessProvider {
	Darkness getDarkness(Instant time, double latitude, double longitude);
}

File: app/src/main/kotlin/se/gustavkarlsson/skylight/android/background/providers/DarknessProvider.kt
package se.gustavkarlsson.skylight.android.background.providers

import org.threeten.bp.Instant

import se.gustavkarlsson.skylight.android.entities.Darkness

interface DarknessProvider {
    fun getDarkness(time: Instant, latitude: Double, longitude: Double): Darkness
}
